sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/BasicRetryPolicy.cs

```diff
@@ -183,7 +189,10 @@ private static bool ShouldRetryException(Exception exception)
         private static TimeSpan CalculateFixedDelay(
             double baseDelaySeconds,
             double baseJitterSeconds,
-            Random random) =>
-            TimeSpan.FromSeconds(baseDelaySeconds + (random.NextDouble() * baseJitterSeconds));
+            Random random)
+        {
+            var delay = baseDelaySeconds + (random.NextDouble() * baseJitterSeconds);
```

Comment:
Technically, validation on the retry options currently prevents this from happening.  It makes sense to apply the clamp, just in case the options limits are changed at some point in the future.